hazelcast/src/main/java/com/hazelcast/util/JsonUtil.java

```diff
@@ -52,7 +52,7 @@ public static int getInt(JsonObject object, String field) {
      */
     public static int getInt(JsonObject object, String field, int defaultValue) {
         final JsonValue value = object.get(field);
-        if (value == null) {
+        if (value == null || value.isNull()) {
```

Comment:
After that change, if `value.isNull = true` then we will return `defaultValue` but it used to be `value.asInt`. Does this introduce any behavior change for parts which uses this util class? cc: @tombujok

Reply:
I am not against this change, but why was this changed? I was under the impression that when we don't need a property we just exclude it. This change would mean that we now accept "null" as value in properties.

Reply:
@tkountis  Before this change, `if` block was meaningless because `null check` was made in the wrong way. I did not want to change current behaviour against the possibility of oversight something so I did not  remove `value == null`. 

yes, with this change, `JsonUtil.getString(mcPhoneHomeInfoJson, "mcLicense", null);` can return `null`.